ui/scss/component/_card.scss

```diff
@@ -662,9 +662,8 @@
 .button_selectedServer {
   display: inline;
   float: right;
-  min-width: var(--side-nav-width--micro);
-  max-width: var(--side-nav-width);
-  margin-left: -10px;
+  min-width: var(--input-select-server-min-width);
```

Comment:
It's good to use variables when the variable is going to be reused multiple places, but this is pretty specific here. 
In this code, probably something like: max-width: 12rem - pick some rem number that is reasonable.
Then shrink down to "mobile" size and see if you need a 
`@media (max-width: $breakpoint-small) { ... }` to adjust.